sdk/typescript/src/dwallet-mpc/globals.ts

```diff
@@ -321,7 +327,7 @@ export async function getNetworkDecryptionKeyID(c: Config): Promise<string> {
 	return decryptionKeyID;
 }
 
-export function cacheNetworkKey(key_id: string, epoch: number, networkKey: Uint8Array) {
+export async function cachePublicParameters(key_id: string, epoch: number, networkKey: Uint8Array) {
```

Comment:
@omersadika this is the only thing I'm not sure of, will this work in a browser?